cohttp-lwt-unix/src/io.ml

```diff
@@ -14,6 +14,8 @@
  *
   }}}*)
 
+open Lwt.Infix
```

Comment:
Is this really needed? It seems that the bind is already defined below, see `let ( >>= ) = Lwt.bind`